modules/geo-layers/src/terrain-layer/terrain-layer.ts

```diff
@@ -343,6 +343,10 @@ export default class TerrainLayer<ExtraPropsT extends {} = {}> extends Composite
       );
     }
 
+    if (!elevationData) {
```

Comment:
Nit: This does seem unrelated...

Reply:
https://github.com/visgl/deck.gl/pull/8290#discussion_r1398842208